src/pages/security/components/_hero_component.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { BackgroundImage, Header } from 'components/elements'
+import { Container } from 'components/containers'
+import device from 'themes/device'
+import { localize } from 'components/localization'
+
+const StyledContainer = styled(Container)`
+    height: 100%;
+    padding: 160px 222px;
+
+    @media ${device.tabletL} {
+        padding: 80px 16px;
+        align-items: flex-start;
+        width: 100%;
+    }
+`
+
+const StyledHeader = styled(Header)`
+    margin-bottom: 1.6rem;
+
+    @media ${device.tabletL} {
+        font-size: 40px;
+        line-height: 50px;
+    }
+`
+
+const Subheadline = styled(Header)`
+    font-weight: normal;
+
+    @media ${device.tabletL} {
+        font-size: 18px;
+        line-height: 26px;
+    }
```

Comment:
Do we need this? I believe "subtitle-1" type should set this automatically on tabletL width?